### DISCUSSION TITLE
Search button for git projects are missing

### DISCUSSION THREAD

**Discussion by cryptolukas** [alefragnani/vscode-project-manager#726] [Q&A] [answered]
Hi,

I moved away from VSCode to vscodium
There is a search button above the git repo list.
How can I get this in vscodium. It is really cool to have it.

**Comment by alefragnani** [comment, marked as answer]
Hi @cryptolukas ,

The extension available for VS Codium is outdated, and not published in sync with VS Code. That´s the reason why it misses a few things available in VS Code.

But I'm planning to start publishing on it as well, so whenever a new version is released, it should be available in both marketplaces.

Stay tuned, it should be out in a couple of weeks

**Comment by cryptolukas** [comment]
Current version for me is v12.8.0
Could you already add it to the vs codium store?
Asking for a friend :rofl:

**Reply by alefragnani** [reply]
Tell your friend that it is already there 😬 

https://open-vsx.org/extension/alefragnani/project-manager

**Comment by cryptolukas** [comment]
Do I have to activate any configuration for it?

**Reply by alefragnani** [reply]
No need to activate any setting. Once you move the mouse over the `Git` panel, it should be visible side by side the `Refresh` and `More Actions... ` buttons:

![image](https://github.com/user-attachments/assets/b954cfbd-ea36-4bce-91d7-8388bbcf5635)

> This printscreen _is not_ from VS Codium, as I don't have it, but it is from the same v12.8.0 release.

**Reply by cryptolukas** [reply]
Project Manager Panel:
![image](https://github.com/user-attachments/assets/053f50c9-479c-4324-ad30-173264a0c770)

Git Panel:
Not available

Explorer Panel:
Not available

How can I make it available on the Explorer Panel?

**Reply by alefragnani** [reply]
I just installed VSCodium 1.91.1 (never used it before) and the extension is working just fine:


https://github.com/user-attachments/assets/27346d29-206b-4c07-a4e8-2898200260cd



Are you _moving/hovering the mouse on that panel_, as I commented above? The buttons only appears when you do this.
